jobs/build/set_client_latest/set-latest.sh

```diff
@@ -18,14 +18,30 @@ TMPDIR=${WORKSPACE}/tools
 mkdir -p "${TMPDIR}"
 cd ${TMPDIR}
 
-# create latest symlink
-ln -svf ${RELEASE} latest
+LESS=$(cat ${RELEASE} |awk -F '.' '{print $1 "." $2 "."  $3 "-0"}')
```

Comment:
Output after `echo` fix:
```
++ echo 4.3.0-0.nightly-2019-11-01-215341
++ awk -F . '{print $1 "." $2 "."  $3 "-0"}'
+ LESS=4.3.0-0-0
```
Suggesting to split on both `.` and `-`
```suggestion
LESS=$(cat ${RELEASE} |awk -F '[.-]' '{print $1 "." $2 "."  $3 "-0"}')
```